drivers/modem/Kconfig.quectel-bg9x

```diff
@@ -0,0 +1,68 @@
+# Quectel BG9x driver options
+
+# Copyright (c) 2020 Analog Life LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config MODEM_QUECTEL_BG9X
+	bool "Enable quectel modem driver"
+	select MODEM_CONTEXT
+	select MODEM_CMD_HANDLER
+	select MODEM_IFACE_UART
+	select MODEM_SOCKET
+	select NET_OFFLOAD
```

Comment:
Is it offloading net_context?

Reply:
Nope, its offloading Net Sockets (NET_SOCKETS_OFFLOAD)..

Reply:
`NET_OFFLOAD` can this be removed?